### PR TITLE
span installation luminaire conditional geography

### PR DESCRIPTION
**Pull request by reno1979**
Bug/aip 8235

### CODE REVIEW

File: src/schema/span-installation/luminaire.repository.ts [hunk 1]
```diff
@@ -45,12 +45,13 @@ export class LuminaireRepository implements ILuminaireRepository {
 		const luminaire = await this.prisma.spanLuminaires.create({ data });
 
 		// Work around Prisma not supporting spatial data types
-		await this.prisma.$executeRaw`
-			UPDATE "spanLuminaires"
-			SET geography = ST_GeomFromGeoJSON(${JSON.stringify(geography)})
-			WHERE id = ${luminaire.id}
-		`;
-
+		if (geography) {
```
Comment (fortunato) [inline src/schema/span-installation/luminaire.repository.ts:48]: Should we make it conditional on the update as well?

**Review by fortunato — CHANGES_REQUESTED** [review]

File: src/schema/span-installation/luminaire.repository.ts (hunk 1 above)
Comment (reno1979) [reply src/schema/span-installation/luminaire.repository.ts:48]: absolutely, I was looking for the file

Comment (fortunato) [reply src/schema/span-installation/luminaire.repository.ts:48]: Same file, different method

Comment (fortunato) [reply src/schema/span-installation/luminaire.repository.ts:48]: Oh wait, update already has the conditional..

**Review by fortunato — APPROVED** [review]

**Pull request merged**